contracts/factory/RegistryFactory.sol

```diff
@@ -56,6 +56,7 @@ contract RegistryFactory is Stakeable, INexusFactory {
 
     function addAttester(address attester) external onlyOwner {
         attesters.push(attester);
+        LibSort.sort(attesters);
```

Comment:
are you sure this works? `LibSort.sort` seems to operate on arrays in memory, you pass a storage array. It would get copied to a memory array, then this memory array gets sorted, but the _storage_ array does not get updated?

If you want to sort the arrays, you might have to do this:

```solidity
address[] memory sorted = attesters;
LibSort.sort(attesters);
attesters = sorted;
```

please write tests to confirm the fixes.

Reply:
yes thought about this yesterday. I need sstore after sorting

Reply:
I will share the changes for review. @MrToph

Reply:
done